sql/plan/ddl_trigger.go

```diff
@@ -164,3 +166,48 @@ func (c *CreateTrigger) RowIter(ctx *sql.Context, row sql.Row) (sql.RowIter, err
 		db: c.db,
 	}, nil
 }
+
+// OrderTriggers is a utility method that first sorts triggers into their precedence. It then splits the triggers into
+// before and after pairs.
+func OrderTriggers(triggers []*CreateTrigger) (beforeTriggers []*CreateTrigger, afterTriggers []*CreateTrigger) {
```

Comment:
Yeah definitely move this back, it's an analyzer function onlyu

Reply:
Resolved in messages